Source/SwiftyCamViewController.swift

```diff
@@ -102,8 +114,11 @@ open class SwiftyCamViewController: UIViewController {
 	public var videoQuality : VideoQuality       = .high
 
 	/// Sets whether flash is enabled for photo and video capture
-
+    @available(*, deprecated, message: "use flashMode .on or .off") //use flashMode
 	public var flashEnabled                      = false
```

Comment:
Make a custom setter here that sets the flashMode to .on / .off that would prevent anyone using the library to stop working